python/pyspark/pandas/frame.py

```diff
@@ -13490,7 +13490,12 @@ def __getattr__(self, key: str) -> Any:
         if key.startswith("__"):
             raise AttributeError(key)
         if hasattr(MissingPandasLikeDataFrame, key):
-            if key in ["asfreq", "asof"] and get_option("compute.pandas_fallback"):
+            if key in [
+                "asfreq",
+                "asof",
+                "convert_dtypes",
+                "infer_objects",
```

Comment:
I think we should add them all for fallback for now without a couple of tests instead of adding each API here with each test cases because we can add the tests in the future, and ideally we should implement all API

Reply:
sg, let me add them in another PR.

Reply:
make a new PR
https://github.com/apache/spark/pull/44965